draft-ietf-httpbis-message-signatures.md

```diff
@@ -413,10 +413,6 @@ The following table presents a non-normative example of metadata values that a s
 
 The Signature Input is a US-ASCII string containing the content that will be signed.  To create it, the signer concatenates together entries for each identifier in the signature's Covered Content in the order it occurs in the list, with each entry separated by a newline `"\n"`.  An identifier's entry is a US-ASCII string consisting of the lowercased identifier followed with a colon `":"`, a space `" "`, and the identifier's canonicalized value (described below).
 
-If Covered Content contains `*created` and the signature's Creation Time is undefined or the signature's Algorithm name starts with `rsa`, `hmac`, or `ecdsa` an implementation MUST produce an error.
```

Comment:
Should not remove this entirely. Implementations should still produce an error if the corresponding header for a signature parameter is not found.

Reply:
Is the suggestion that this constraint be applied regardless of algorithm name?

I completely agree that this "algorithm name starts with" language is hugely problematic.

Reply:
@llorllale @jricher right! Both of you are correct! Fixing...

Reply:
@jricher 

> Is the suggestion that this constraint be applied regardless of algorithm name?

Yes.

Reply:
Fixed!